dockertest/environment.py

```diff
@@ -62,3 +65,13 @@ def get_selinux_context(path):
     """
     # First list item is null-terminated string length
     return selinux.getfilecon(path)[1]
+
+
+def selinux_is_enforcing():
+    """
+    Return True if selinux is currently in enforcing mode.
+    """
+    # Travis complains about missing selinux.ENFORCING
+    # pylint: disable=C0103
+    ENFORCING = 1  # Travis complains about missing selinux.ENFORCING
+    return selinux.security_getenforce() == ENFORCING
```

Comment:
I don't think I understand your emoji comment. 

What I'm suggesting is: since the documentation uses 0 and 1, and since anyone who has ever run `setenforce` on the command line already knows about 0 and 1, I think it's safe to assume that 0/1 are known conventions and it's OK to push them up one level. Using `failif_ne` instead of `failif` has a big advantage in that the actual and expected values are displayed right there in the failure message, making it easy for a maintenance programmer to see exactly what happened.